multimodal/health_multimodal/text/data/io.py

```diff
@@ -0,0 +1,47 @@
+#  -------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  -------------------------------------------------------------------------------------------
+
+import logging
+from typing import Any, List, Union
+
+from transformers import BertTokenizer
+
+
+class TextInput:
+    """
+    Text input class that can be used for inference and deployment.
+    Implements tokenizer related operations and ensure that input strings
+    conform with the standards expected from a BERT model.
```

Comment:
The summary should go in the first line, followed by a blank line:

https://github.com/microsoft/hi-ml/blob/9fc160364f3d47a414dd563c912f034ec4ba05f2/hi-ml-histopathology/src/histopathology/utils/girder.py#L112-L122